tests/_support/Step/Acceptance/Administrator/Login.php

```diff
@@ -20,6 +20,7 @@ public function loginIntoJoomlaAdministrator($username, $password)
 
 	/**
 	 * @Then I should see the administrator dashboard
+	 * @When I see the administrator dashboard
```

Comment:
We could add a regex here or just add the two gherkin sentences that will call the Step method. See: http://codeception.com/docs/07-BDD#step-definitions

I took the option of adding the two sentences.